tensorboard/data/experimental/BUILD

```diff
@@ -10,13 +10,13 @@ exports_files(["LICENSE"])
 py_library(
     name = "base_experiment",
     srcs = ["base_experiment.py"],
-    srcs_version = "PY3",
+    srcs_version = "PY2AND3",
```

Comment:
Why this change? It doesn’t look like there are any Google-internal
reverse dependencies of `//tensorboard:lib` that are still Python 2
binaries. This change is okay with me if it’s really necessary for some
reason, but in general we’ve been preferring to write new code as py3.

Reply:
Yeah, see the comment I sent yesterday. This change is necessary because some internal build targets are PY2. Specifically the notebook-related builds. I discovered breakages in those builds during a test sync of this PR, which prompted this change.

Reply:
After some further testing, I confirmed that the `PY2AND3` part isn't necessary. I have pushed a new commit to change them all to `PY3` in this file. PTAL @wchargin